syntax = "proto2";

package opentxs.proto;
option optimize_for = LITE_RUNTIME;

import public "PeerEnums.proto";
import public "PeerReply.proto";
import public "PeerRequest.proto";

message PeerObject {
    optional uint32 version = 1;
    optional PeerObjectType type = 2;
    optional string otmessage = 3;
    optional PeerRequest otrequest = 4;
    optional PeerReply otreply = 5;
}
